src/App.js

```diff
@@ -14,11 +14,37 @@ function useSemiPersistentState() {
   }, [todoList]);
 
   return [todoList, setTodoList];
-}
+}*/
 
 const App = () => {
   // const savedList = JSON.parse(localStorage.getItem("savedTodoList"));
-  const [todoList, setTodoList] = useSemiPersistentState();
+  // const [todoList, setTodoList] = useSemiPersistentState();
+
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    new Promise(function (resolve, reject) {
+      setTimeout(
```

Comment:
remember to set the delay time (2000 milliseconds) -- [here's a link if you need more info](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout)